.github/actions/install-aiida-core/action.yml

```diff
@@ -0,0 +1,50 @@
+name: Install aiida-core
+description: Install aiida-core package and its Python dependencies
+
+inputs:
+  python-version:
+    description: Python version
+    default: '3.9'              # Lowest supported version
+    required: false
+  extras:
+    description: aiida-core extras (including brackets)
+    default: ''
+    required: false
+  # NOTE: Hard-learned lesson: we cannot use type=boolean here, apparently :-(
+  # https://stackoverflow.com/a/76294014
```

Comment:
Now _that_ is counter-intuitive I agree

Reply:
Thank you! I really needed this validation from somebody :smiling_face_with_tear: